tools/c7n_azure/tests/test_app_service_plan.py

```diff
@@ -101,12 +104,17 @@ def test_resize_plan_linux(self):
         resources = p.run()
         self.assertEqual(1, len(resources))
 
+        name, args, kwargs = update_mock.mock_calls[0]
+        self.assertEqual('cctest-appserviceplan-linux', args[1])
+        self.assertEqual('F1', args[2].sku.name)
+        self.assertEqual('FREE', args[2].sku.tier)
+
+    @patch('azure.mgmt.web.operations.app_service_plans_operations.'
+           'AppServicePlansOperations.update')
     @arm_template('appserviceplan.json')
-    def test_resize_plan_from_resource_tag(self):
-        web_management_client = self.session.client(
-            'azure.mgmt.web.WebSiteManagementClient')
-        app_plan = web_management_client.app_service_plans.get('test_appserviceplan',
-                                                               'cctest-appserviceplan-win')
+    def test_resize_plan_from_resource_tag(self, update_mock):
+        app_plan = self.client.app_service_plans.get('test_appserviceplan',
```

Comment:
We can add tag in template and we don't need this check too since we never actually change the sku